Source/Core/Core/PowerPC/JitArm64/JitArm64_FloatingPoint.cpp

```diff
@@ -355,7 +357,8 @@ void JitArm64::fcmpx(UGeckoInstruction inst)
 
 	FixupBranch pNaN1, pNaN2, pNaN3, pLesser, pGreater;
 	FixupBranch continue1, continue2, continue3;
-	ORR(XA, ZR, 32, 0, true);
+
+	m_float_emit.ORR(XA, ZR, 32, 0, true);
```

Comment:
Yet they would kind of have to for this line:
- https://github.com/dolphin-emu/dolphin/blob/696d6eae099feba23c280017c5184785fc1e3076/Source/Core/Core/PowerPC/JitArm64/JitArm64_FloatingPoint.cpp#L446

And for this line:
- https://github.com/dolphin-emu/dolphin/blob/696d6eae099feba23c280017c5184785fc1e3076/Source/Core/Core/PowerPC/JitArm64/JitArm64_Paired.cpp#L178

no?